src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20180607_master_disk_size.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I think this file needs to be referenced here to actually run:

https://github.com/DataBiosphere/leonardo/blob/develop/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changelog.xml